examples/mixture_models/dirichlet_mixture_of_multinomials.ipynb

```diff
@@ -0,0 +1,1815 @@
+{
```

Comment:
<ul><li>Nitpick: I'd call the concentration parameter... <code>concentration</code> , instead of <code>alpha</code>, which is more obscure</li><li>"In this iteration of our model we'll explicitly include the latent multinomial probability...": this is confusing: there is no <code>\pi</code> parameter in the model below. Didn't you rather mean:</li></ul><blockquote>In this iteration of our model we'll explicitly include the latent multinomial probability <code>p_i</code> , which models the probabilities of each species in each forest, and that we would never observe in the real world.</blockquote><p>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Actually the better would be to use the same parameter names you mentioned in the beginning when you generated our data, although then you have the problem of shadowing the original variables (I always struggle with this!). I usually end up doing <code>true_frac = ...</code>  when generating data, and in the models call it <code>frac</code>  <span class="ql-cursor">﻿</span></p>

Reply:
<p>I've renamed simulation and model variables throughout. See my comment above.</p>